src/app/components/osd/osd.component.ts

```diff
@@ -185,11 +189,14 @@ export class OsdComponent implements AfterViewInit, OnDestroy {
 
         this.viewer.addHandler('page', ({ page }) => {
           this.pageChange.next(page + 1);
+          this._page = page;
```

Comment:
You should never change `_page` explicitly` it will create different values between pageChange and the actual page. You should use the event emitter IMHO